napari/utils/events/evented_model.py

```diff
@@ -139,23 +175,21 @@ class Config:
 
     _deps = getattr(cls.__config__, 'dependencies', None)
     if _deps:
-        for prop, fields in _deps.items():
-            if prop not in cls.__property_setters__:
+        for prop_name, fields in _deps.items():
+            if prop_name not in cls.__property_setters__:
                 raise ValueError(
                     'Fields with dependencies must be property.setters. '
-                    f'{prop!r} is not.'
+                    f'{prop_name!r} is not.'
                 )
             for field in fields:
                 if field not in cls.__fields__:
                     warnings.warn(f"Unrecognized field dependency: {field}")
-                deps.setdefault(field, set()).add(prop)
+                deps.setdefault(field, set()).add(prop_name)
```

Comment:
Maybe some test to cover this? 
EventedModel is the core of napari so I would like to see 100% coverage